test/stripe/api_operations_test.rb

```diff
@@ -38,8 +38,6 @@ class MainResource < APIResource
         OBJECT_NAME = "mainresource".freeze
         nested_resource_class_methods :nested,
                                       operations: %i[create retrieve update delete list]
-
-        OBJECT_NAME = "mainresource".freeze
```

Comment:
Noticed a weird warning today. This likely should have been caught by the test suite though so it's worth investigating what caused it to not be seen though it's just in tests.

Reply:
Arg yes, I've run into this type of thing before.

Unfortunately, there doesn't seem to be any [easy way to have Ruby treat warnings as errors](https://stackoverflow.com/questions/660737/can-you-ask-ruby-to-treat-warnings-as-errors) (I don't really want to monkeypatch `Kernel` although I suppose we could consider it).

These don't come up that often though, so I'm not sure it's a huge problem.